static/js/common/upload-addon.js

```diff
@@ -337,11 +337,14 @@
                       // Specific messages for unlisted addons.
                       var isSideload = $('#id_is_sideload').is(':checked') || $new_form.data('addon-is-sideload');
                       var automaticValidation = $('#create-addon').data('automatic-validation') || $new_form.data('automatic-validation');
+                      var passedValidation = (!v.signing_summary || (!v.signing_summary.low &&
+                                                                     !v.signing_summary.medium &&
+                                                                     !v.signing_summary.high));
                       if (!isListed) {
                         if (isSideload) {
                           $("<p>").text(gettext("Your submission will go through a manual review.")).appendTo(upload_results);
                         } else {
-                          if (warnings === 0) {
+                          if (passedValidation) {
```

Comment:
The signing summary should always be there (now that we use the updated version of the amo-validator). And for automatic validation to succeed, we only check if there's no signing related messages (we don't care about the trivial ones, but we do fail the auto validation if there's any low/medium/high security warnings).

The current messages/warnings that were also meant to fail the automatic validation now have a signing severity attached to them.